### PR TITLE
Dont mutate the hash map

### PR DESCRIPTION
**Pull request by wrobstory**
We mutate the hashmap when we get deps, so this removes that mutation. 

Also I guess we dont automatically run `cargo fmt`, and I did, and now we have many changes :sob:

### CODE REVIEW

File: crates/driver/src/print_build.rs [hunk 1]
```diff
@@ -476,10 +476,7 @@ where
     apply_attr_string_lists(&mut extra_kv_pairs, &graph_node.node_metadata);
     // before we give extra_kv_pairs away to make the main target,
     // we need to clone deps here for a later use in secondaries.
-    let deps = extra_kv_pairs
-                .entry("deps".to_string())
-                .or_default()
-                .clone();
+    let deps = extra_kv_pairs.get("deps").cloned().unwrap_or_else(Vec::new);
```
Comment (wrobstory) [inline crates/driver/src/print_build.rs:479]: This is the only *actual* change, the rest is whitespace

**Comment by wrobstory** [comment]
Errrrr im sure I updated the tests, but i dont see those updates, one sec

File: crates/driver/src/print_build.rs [hunk 2]
```diff
@@ -910,16 +918,16 @@ filegroup(
 proto_library(
     name='protos',
     srcs=[':protos_files'],
-    deps=[],
```
Comment (wrobstory) [inline crates/driver/src/print_build.rs:913]: Ok, one test change here

File: crates/driver/src/print_build.rs [hunk 3]
```diff
@@ -938,7 +946,6 @@ filegroup(
...
```
Comment (wrobstory) [inline crates/driver/src/print_build.rs:941]: and here

**Review by eed3si9n — APPROVED** [review]
Thanks @wrobstory!

**Comment by eed3si9n** [comment]
Error: Generated example/src/main/protos/com/example/BUILD.bazel differs from the checked-in version
diff --git a/example/src/main/protos/com/example/BUILD.bazel b/example/src/main/protos/com/example/BUILD.bazel
index 5fe4fbd..3cde8f3 100644

```diff
--- a/example/src/main/protos/com/example/BUILD.bazel
+++ b/example/src/main/protos/com/example/BUILD.bazel
@@ -1,6 +1,6 @@
 load('@rules_proto//proto:defs.bzl', 'proto_library')
 load('@rules_python//python:proto.bzl', 'py_proto_library')
 filegroup(name='example_files', srcs=glob(include=['**/*.proto']), visibility=['//visibility:public'])
-proto_library(name='example', srcs=[':example_files'], deps=[], visibility=['//visibility:public'])
+proto_library(name='example', srcs=[':example_files'], visibility=['//visibility:public'])
```

**Pull request merged**